### ISSUE TITLE
after npm installation it fails to build on android

### ISSUE THREAD

**Issue by raajjemv** [TheWidlarzGroup/react-native-video#2601]
# Bug

info Running jetifier to migrate libraries to AndroidX. You can disable it using "--no-jetifier" flag.
Jetifier found 1231 file(s) to forward-jetify. Using 10 workers...
info JS server already running.
info Installing the app...
WARNING:: The specified Android SDK Build Tools version (29.0.2) is ignored, as it is below the minimum supported version (30.0.2) for Android Gradle Plugin 4.2.2.
Android SDK Build Tools 30.0.2 will be used.
To suppress this warning, remove "buildToolsVersion '29.0.2'" from your build.gradle file, as each version of the Android Gradle Plugin now has a default version of the build tools.
> Task :app:checkDebugAarMetadata FAILED

Deprecated Gradle features were used in this build, making it incompatible with Gradle 8.0.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

See https://docs.gradle.org/7.2/userguide/command_line_interface.html#sec:command_line_warnings
11 actionable tasks: 2 executed, 9 up-to-date

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:checkDebugAarMetadata'.
> Could not resolve all files for configuration ':app:debugRuntimeClasspath'.
   > Could not find com.yqritc:android-scalablevideoview:1.0.4.
     Searched in the following locations:
       - file:/Users/imsamaah/Apps/raajjemv/node_modules/react-native/android/com/yqritc/android-scalablevideoview/1.0.4/android-scalablevideoview-1.0.4.pom
       - file:/Users/imsamaah/Apps/raajjemv/node_modules/jsc-android/dist/com/yqritc/android-scalablevideoview/1.0.4/android-scalablevideoview-1.0.4.pom
       - https://repo.maven.apache.org/maven2/com/yqritc/android-scalablevideoview/1.0.4/android-scalablevideoview-1.0.4.pom
       - https://dl.google.com/dl/android/maven2/com/yqritc/android-scalablevideoview/1.0.4/android-scalablevideoview-1.0.4.pom
       - https://www.jitpack.io/com/yqritc/android-scalablevideoview/1.0.4/android-scalablevideoview-1.0.4.pom
     Required by:
         project :app > project :react-native-video

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 9s

error Failed to install the app. Make sure you have the Android development environment set up: https://reactnative.dev/docs/environment-setup.
Error: Command failed: ./gradlew app:installDebug -PreactNativeDevServerPort=8081

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:checkDebugAarMetadata'.
> Could not resolve all files for configuration ':app:debugRuntimeClasspath'.
   > Could not find com.yqritc:android-scalablevideoview:1.0.4.
     Searched in the following locations:
       - file:/Users/imsamaah/Apps/raajjemv/node_modules/react-native/android/com/yqritc/android-scalablevideoview/1.0.4/android-scalablevideoview-1.0.4.pom
       - file:/Users/imsamaah/Apps/raajjemv/node_modules/jsc-android/dist/com/yqritc/android-scalablevideoview/1.0.4/android-scalablevideoview-1.0.4.pom
       - https://repo.maven.apache.org/maven2/com/yqritc/android-scalablevideoview/1.0.4/android-scalablevideoview-1.0.4.pom
       - https://dl.google.com/dl/android/maven2/com/yqritc/android-scalablevideoview/1.0.4/android-scalablevideoview-1.0.4.pom
       - https://www.jitpack.io/com/yqritc/android-scalablevideoview/1.0.4/android-scalablevideoview-1.0.4.pom
     Required by:
         project :app > project :react-native-video

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 9s

    at makeError (/Users/imsamaah/Apps/raajjemv/node_modules/@react-native-community/cli-platform-android/node_modules/execa/index.js:174:9)
    at /Users/imsamaah/Apps/raajjemv/node_modules/@react-native-community/cli-platform-android/node_modules/execa/index.js:278:16
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async runOnAllDevices (/Users/imsamaah/Apps/raajjemv/node_modules/@react-native-community/cli-platform-android/build/commands/runAndroid/runOnAllDevices.js:109:5)
    at async Command.handleAction (/Users/imsamaah/Apps/raajjemv/node_modules/@react-native-community/cli/build/index.js:192:9)
info Run CLI with --verbose flag for more details.

## Platform
It happens only in android. in IOS it works fine

Which player are you experiencing the problem on:

* Android ExoPlayer

## Environment info

System:
    OS: macOS 12.1
    CPU: (10) arm64 Apple M1 Pro
    Memory: 264.20 MB / 16.00 GB
    Shell: 5.8 - /bin/zsh
  Binaries:
    Node: 17.5.0 - /opt/homebrew/bin/node
    Yarn: Not Found
    npm: 8.4.1 - /opt/homebrew/bin/npm
    Watchman: 2022.02.14.00 - /opt/homebrew/bin/watchman
  Managers:
    CocoaPods: 1.11.2 - /usr/local/bin/pod
  SDKs:
    iOS SDK:
      Platforms: DriverKit 21.2, iOS 15.2, macOS 12.1, tvOS 15.2, watchOS 8.3
    Android SDK:
      API Levels: 23, 25, 28, 29, 30
      Build Tools: 29.0.2, 30.0.2
      System Images: android-29 | ARM 64 v8a, android-29 | Google APIs ARM 64 v8a, android-32 | Google APIs ARM 64 v8a
      Android NDK: Not Found
  IDEs:
    Android Studio: 2020.3 AI-203.7717.56.2031.7935034
    Xcode: 13.2.1/13C100 - /usr/bin/xcodebuild
  Languages:
    Java: 11.0.11 - /usr/bin/javac
  npmPackages:
    @react-native-community/cli: Not Found
    react: 17.0.2 => 17.0.2
    react-native: 0.67.2 => 0.67.2
    react-native-macos: Not Found
  npmGlobalPackages:
    *react-native*: Not Found
React native info output:

```bash
 // paste it here
```

Library version: x.x.x

## Steps To Reproduce

1.npm install --save react-native-video
2.npx react-native run-android
...

**Comment by ThomasStubbe** [comment]
Same issue: it's because jcenter is deprecated and removed from the react-native demo projects (or when you update react-native with the upgrade-helper).

I solved it by adding jcenter() back to android\build.gradle under `allprojects { ... repositories { ... jcenter() ... } } `

**Comment by NensiKasundra** [comment]
Yes Same Issue. 

Execution failed for task ':app:mergeDebugJavaResource'.
> Could not resolve all files for configuration ':app:debugRuntimeClasspath'.
   > Could not find com.yqritc:android-scalablevideoview:1.0.4.


jcenter is deprecated. it's not permanent solution. 

It's not available on maven.

- https://dl.google.com/dl/android/maven2/com/yqritc/android-scalablevideoview/1.0.4/android-scalablevideoview-1.0.4.pom
- https://repo.maven.apache.org/maven2/com/yqritc/android-scalablevideoview/1.0.4/android-scalablevideoview-1.0.4.pom
- https://www.jitpack.io/com/yqritc/android-scalablevideoview/1.0.4/android-scalablevideoview-1.0.4.pom


Please Update.

**Comment by Pavel-Ananich** [comment]
The same issue

**Comment by udayarajurs** [comment]
> Same issue: it's because jcenter is deprecated and removed from the react-native demo projects (or when you update react-native with the upgrade-helper).
> 
> I solved it by adding jcenter() back to android\build.gradle under `allprojects { ... repositories { ... jcenter() ... } } `

this working for me

**Comment by jason-vault** [comment]
conversely, you can be more explicit with:
```
...
allprojects {
    repositories {
       ... other repositories
        jcenter() {
            content {
                includeModule("com.yqritc", "android-scalablevideoview")
            }
        }
    }
}
```

**Comment by arsimr16** [comment]
I was able to use `react-native-video 5.2.0` without using jcenter by switching to the Android ExoPlayer.  `android-scalablevideoview` is only used by the Android MediaPlayer which is being removed in version 6.x of react-native-video anyway.

1) Add the following lines to settings.gradle to use the ExoPlayer instead of the MediaPlayer:
```
include ':react-native-video'
project(':react-native-video').projectDir = new File(rootProject.projectDir, '../node_modules/react-native-video/android-exoplayer')
```

2) Once I switched to the ExoPlayer, I got errors from `react-native-video/android-exoplayer/build.gradle` about `com.google.android.exoplayer:exoplayer:2.13.2` and `com.google.android.exoplayer:extension-okhttp:2.13.2` not being available.  I made a patch to change both versions to `2.13.3`, and my app built successfully after that.
